### PR TITLE
Nessie: Fix testcase failures

### PR DESCRIPTION
**Pull request by ajantha-bhat**
https://github.com/apache/iceberg/pull/6789 was merged recently and the PR didn't rebase before merging as there was no conflict.

But there was another PR that got merged https://github.com/apache/iceberg/pull/7146 which enforces the namespace creation. Hence the new tests failed. 

Apologies for breaking the master build!

### CODE REVIEW

**Comment by ajantha-bhat** [comment]
cc: @Fokko, @nastra

**Review by dimas-b — APPROVED** [review]

File: nessie/src/test/java/org/apache/iceberg/nessie/TestMultipleClients.java [hunk 1]
```diff
@@ -98,12 +98,12 @@ public void testLoadNamespaceMetadata() {
 
   @Test
   public void testListTables() {
-    catalog.createTable(TableIdentifier.parse("foo.tbl1"), schema);
+    createTable(catalog, TableIdentifier.parse("foo.tbl1"), schema);
```
Comment (nastra) [inline nessie/src/test/java/org/apache/iceberg/nessie/TestMultipleClients.java:101]: why not just call `catalog.createNamespace(Namespace.of("foo"), Collections.emptyMap());` at the beginning of these methods?

**Review by nastra — CHANGES_REQUESTED** [review]

File: nessie/src/test/java/org/apache/iceberg/nessie/TestMultipleClients.java [hunk 2]
```diff
@@ -98,12 +98,12 @@ public void testLoadNamespaceMetadata() {
...
     Assertions.assertThat(catalog.listTables(Namespace.of("foo")))
         .containsExactlyInAnyOrder(TableIdentifier.parse("foo.tbl1"));
 
     // another client creates a table with the same nessie server
-    anotherCatalog.createTable(TableIdentifier.parse("foo.tbl2"), schema);
+    createTable(anotherCatalog, TableIdentifier.parse("foo.tbl2"), schema);
```
[the remark this answers is not in the document]
Comment (ajantha-bhat) [reply nessie/src/test/java/org/apache/iceberg/nessie/TestMultipleClients.java]: yeah, creating again from a different client is not required as namespace already exist.  I removed it.

**Review by nastra — APPROVED** [review]

**Review by Fokko — APPROVED** [review]
Thanks for fixing this so quickly @ajantha-bhat

**Comment by ajantha-bhat** [comment]
PR is ready for merging.

**Comment by Fokko** [comment]
Thanks @stevenzwu for merging :)

**Comment by stevenzwu** [comment]
merge this to fix the master branch build failure. thanks @ajantha-bhat for the fix and @nastra @Fokko @dimas-b for the review

**Pull request merged**